### ISSUE TITLE
Global Nav accessibility (34)

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#444]
The a11y team said the following had issues in the global nav found at 
- Tab to anchors and menu triggers
- always visually show where the focus is
- arrows only to get through menu items
- tab to move from a menu item to the next item _after_ the trigger
- menu isn't closing when tabbing away from it
- aria-haspopup on the menu trigger, not on the `<li>`
- clicking menu item is supposed to navigate to the page
- clicking hte down arrow is supposed to open the menu

I'm not 100% sure these are the agreed upon UX patterns for a11y. He said it was a 30 sec glance. I can confirm that the menu doesn't close on tab off. The `aria-haspopup="true"` though should be on the button as per SLDS markup, so take the above with a grain of salt.

**Comment by interactivellama** [comment]
De-prioritized until global nav use comes back from the grave. @tweettypography
